Dockerfile

```diff
@@ -4,11 +4,25 @@ ARG runtime
 # Install dev dependencies
```

Comment:
Is this the Dockerfile that will be used to ship `datadog-lambda-rb` to customers, or is it just for development?

I ask because we are creating files (`gem install ddtrace -v 1.11.0`) then removing them (`rm -rf ./ruby/gems/$runtime/gems/debase-ruby_core_source*/`) in a separate docker `RUN` step, which keeps all the files generated by `gem install ddtrace -v 1.11.0` in the docker image anyway, only to be deleted by the `rm -rf ./ruby/gems/$runtime/gems/debase-ruby_core_source*/` step. This means the docker image is still as large as it can be, despite the `rm` calls.

Reply:
Yes, this Dockerfile is used by the `build_layers.sh` script in order to install `datadog-lambda-rb` and its dependencies. Further, the same script gets the output of the Dockerfile to zip it. 

I am aware of the docker image still being really large, but after we zip it and get its output, the file does not contain the removed files.

I'm not sure if I should be doing something else here, since the current behavior does delete the gem we don't need. Any thoughts on this?